demos/slider.scss

```diff
@@ -17,3 +17,14 @@
 @import "./common";
 @import "../packages/mdc-slider/mdc-slider";
 @import "../packages/mdc-typography/mdc-typography";
+@import "../packages/mdc-theme/color-palette";
+
+.demo-slider--custom {
+  $main-color: $material-color-red-700;
+  $tick-color: $material-color-red-300;
+
+  @include mdc-slider-rail-color($main-color);
```

Comment:
Given that it seems to be ridiculously common to set all of these to the same color, should we have a mixin that sets all of them? Maybe takes a color + an ink color for the popup indicator?

Reply:
Done!

I decided to keep it simple and pick the ink color automatically, just like `mdc-slider-value-pin-fill-color-accessible`.

The new combined mixin is `mdc-slider-color-accessible`.